src/configuration/notation.ts

```diff
@@ -32,6 +33,7 @@ export class Notation {
       this.isSurroundedByAngleBrackets(key) &&
       key !== '<BS>' &&
       key !== '<SHIFT+BS>' &&
```

Comment:
Why are both `<Shift+BS>` and `<S-BS>` here? Can't we normalize to the latter?

Reply:
Well when I made this change I didn't really had much time to see if it would mess up anything by removing that, so I left it there. But from what I've looked now I think it can be removed. It might create some issues with existing tests but we can update the tests. I'll try to look into it.

Reply:
I think I broke the tests in the same way, but haven't had time to go back and fix it. I'd love if we could just normalize on the way in, have a canonical representation (`<S-___>` in this case) internally, and not worry about it.

Reply:
That is usually what happens. But in tests sometimes we call the `modeHandler.handleKeyEvent('someKey')` directly and that doesn't go through normalization since that is done on the 'configuration' when loading the boundKeys or when handling remaps on the 'remappingValidator'.

Reply:
Right, I think we should just make sure we're using the normalized representations in the test suite

Reply:
I've just pushed a fix that changes all occurrences of `<shift+tab>` and `<shift+BS>` to `<S-tab>` and `<S-bs>`, respectively.